public/app/features/alerting/unified/components/amroutes/MuteTimingForm.tsx

```diff
@@ -22,12 +22,14 @@ import { createMuteTiming, defaultTimeInterval } from '../../utils/mute-timings'
 import { initialAsyncRequestState } from '../../utils/redux';
 import { AlertManagerPicker } from '../AlertManagerPicker';
 import { AlertingPageWrapper } from '../AlertingPageWrapper';
+import { ProvisioningAlert } from '../Provisioning';
 
 import { MuteTimingTimeInterval } from './MuteTimingTimeInterval';
 
 interface Props {
   muteTiming?: MuteTimeInterval;
   showError?: boolean;
+  provenance?: string;
```

Comment:
The provenance for anything that has been provisioned indicates what tool or how it was provisioned. Currently if you use the API it will be set to `api`, when using the terraform provider it might be `terraform` and for file provisioning it might be `file` or `config`.